manifests/entandoapps.entando.org.crd.yaml

```diff
@@ -74,44 +58,33 @@ spec:
                   type: string
                 ingressHostName:
                   description: >
-                    The hostname that will be used to access this EntandoApp  from outside the cluster. The Entando
+                    The hostname that will be used to access this EntandoApp from outside the cluster. The Entando
                     Operator will create an Ingress reflecting this hostname, and expose this EntandoApp on the
                     path specified in 'EntandoApp.spec.ingressPath' on
-                    this host. If omitted, the Entando Operator will automatically generate a hostname using the value of
-                    the ENTANDO_DEFAULT_ROUTING_SUFFIX environment variable that the Entando Operator was configured with.
-                    It will prefix the name and namespace of the EntandoApp to this default routing suffix.
+                    this host. If omitted, the Entando Operator will automatically generate a hostname.
+
+                    See [here] (https://dev.entando.org/v6.3/docs/concepts/operator-intro.html#ingress-path-generation) for how the operator will create and configure the Ingress.
```

Comment:
We're going to have to keep in mind we have version-specific links in the CRDs. Do we have a README or doc somewhere that documents our release instructions for the operator? We might need a reminder there to review these kinds of links as we're releasing new versions of the operator. I have something like that in the entando-docs README which I review when we cut a new version.